rental.rb

```diff
@@ -0,0 +1,24 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+
+    @person = person
+    person.rentals << self
+
+    @book = book
+    @book.rentals << self
+  end
+
+  def book=(book)
+    @book = book
+    book.rentals << self
+  end
+
+  def person=(person)
+    @person = person
+    person.rentals << self
+  end
```

Comment:
- Hey, :wave: , you really do not need to add each of these methods, remember each of those classes has a method related to all these you've added. Instead, what you want to do is to update your constructor to handle all of those for you, 

```ruby
def initialize(date, person, book)
    @date = date
    @person = person
    @book = book
    person.rentals << self
    book.rentals << self
  end
```

This establishes all of those as each class also has the add_rentals method. let's update this.